src/components/ImageSelector.js

```diff
@@ -8,6 +8,24 @@ import styles from '../styles';
 import './ImageSelector.css';
 
 const ImageSelector = props => {
+  var TextFieldHolder = [];
+  var length = props.imageNames.length;
+  for (let i = 0; i < length; i++) {
+    TextFieldHolder.push(
+      <div>
+        <TextField
+          floatingLabelText={constants.IMAGE_ID_FLOATING_LABEL}
+          floatingLabelFixed={true}
+          hintText={constants.IMAGE_ID_HINT_TEXT}
+          errorText={props.imageIdErrorText}
+          value={props.imageNames[i]}
+          onChange={(e, newValue) => {
+            props.handleChange(constants.IMAGE_ID_IDENTIFIER, e, newValue, i);
```

Comment:
`handleChange` hasn't been updated yet.

Reply:
I think if there are more than 1 image, you can probably remove TextField. But now there is no text field shown by default, unless someone uploads something?

Reply:
Also, I would like to put some limit on number of multiple uploads. I think 10 should be good enough, otherwise if we upload so many images and the promise fails in between or some network error. All of the uploads will be useless.

Reply:
Yes now I have set image limit to be 10. And no TextField when no image is selected.

Reply:
What about `handleChange`?